src/crypto/public_key/elliptic/ecdh.js

```diff
@@ -340,10 +353,50 @@ async function ellipticPublicEphemeralKey(curve, Q) {
   return { publicKey, sharedKey };
 }
 
+/**
+ * Generate ECDHE secret from private key and public part of ephemeral key using nodeCrypto
+ *
+ * @param  {Curve}                  curve          Elliptic curve object
+ * @param  {Uint8Array}             V              Public part of ephemeral key
+ * @param  {Uint8Array}             d              Recipient private key
+ *
+ * @returns {Promise<{secretKey: Uint8Array, sharedKey: Uint8Array}>}
```

Comment:
I agree that these extra empty lines make it nice and tidy, but, these comments are already very large, and personally I almost never look at them, so it's a bit a wasted space. Also I think it's better to improve the parameter names instead of improving these comments (don't have to do that now, though). It's a bit nitpicky but could you remove the empty lines?